cypress/automated-tests/youth-pass/eligibilty-checker-blockers.spec.js

```diff
@@ -0,0 +1,98 @@
+const faker = require('faker');
+
+describe('Eligibility Checker Blockers', () => {
+    const youthPassUrl = Cypress.env('youth_pass_url');
+    const eligibleZipCode = '02114';
+    
+    it('enters an ineligible zip code', () => {
+        const ineligibleZipCode = '01545';
+
+        cy.visit(youthPassUrl);
+        cy.get('#form-section-0 > .form-section-buttons > .form-section-next').click();
+        cy.get('#form-element-wrapper_182').should('not.be.visible');
+
+        cy.get('#element105').type(ineligibleZipCode).blur();
+        cy.get('#form-element-wrapper_182').should('be.visible');
+        cy.get('#form-section-1 > .form-section-buttons > .form-section-next').should('not.be.visible');
+    });
+    
+    it('enters an age below 12', () => {
+        const today = new Date();
```

Comment:
Keeping these variables scoped as tightly as possible. I can move them up to the `describe` block if that's more closely aligned with CTD testing style, however

Reply:
This makes sense to me. If you used `today` in the next test as well maybe that one could be moved out, but otherwise these seem specific to this test.